processing/src/main/java/io/druid/query/aggregation/JavaScriptAggregatorFactory.java

```diff
@@ -40,7 +40,7 @@
 import java.util.Comparator;
 import java.util.List;
 
-public class JavaScriptAggregatorFactory implements AggregatorFactory
+public class JavaScriptAggregatorFactory extends AggregatorFactory
```

Comment:
I think we do want to support some schema evolution, so maybe the right answer is we actually should allow the merge, and the fnCombine from the higher-versioned-segment should be used?